testsuite/src/parser_evaluator.rs

```diff
@@ -135,6 +137,36 @@ pub fn parse_w3c_rdf_test_file(
     }
 }
 
+#[cfg(feature = "generalized")]
+pub fn parse_w3c_rdf_test_file_with_gtrig(
```

Comment:
This function name seems a bit misleading because the gtrig parser is only used on .trig file.

Reply:
renaming to `parse_w3c_rdf_test_file_for_gtrig`